artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
Yes, I'm reusing this branch :+1:  
Right now, I'm searching for an already existing example on CLI commands to look on how it is implemented (and use the same name pattern) as a reference, but i haven't found it yet, do you know where it is?